tapchannel/aux_traffic_shaper.go

```diff
@@ -320,6 +327,12 @@ func (s *AuxTrafficShaper) paymentBandwidth(htlc *rfqmsg.Htlc,
 				"ID %x (SCID %d)", rfqID[:], rfqID.Scid())
 		}
 
+		// If the channel peer does not match the quote peer, continue
+		// to the next available quote.
+		if !bytes.Equal(quotePeer[:], peer[:]) {
```

Comment:
See my comment above. Can just use `quotePeer route.Vertex`, then use the `==` comparison.